controllers/templatesync/template_sync.go

```diff
@@ -999,6 +1016,8 @@ func (r *PolicyReconciler) cleanUpExcessTemplates(
 			}
 			// Iterate over the ConstraintTemplates to gather the Constraints on the cluster
 			for _, gkCT := range gkConstraintTemplateListv1.Items {
+				gkCT := gkCT
```

Comment:
Oh? what is this

Reply:
The linter was failing because in Go < 1.22, the same memory address used for each item in the range so getting a pointer of `gkCT` without reassigning it will lead to strange results.

I'm not sure that this particular code would cause an issue though, but as I said, the linter started failing so I made it not fail.